tests/sqlx/lexer.spec.ts

```diff
@@ -1,92 +1,15 @@
-import { parseSqlx, SyntaxTreeNode, SyntaxTreeNodeType } from "@dataform/sqlx/lexer";
+import { SyntaxTreeNode, SyntaxTreeNodeType } from "@dataform/sqlx/lexer";
 import { suite, test } from "@dataform/testing";
 import { expect } from "chai";
 
 suite("@dataform/sqlx", () => {
-  suite("outer SQL lexing", () => {
-    test("backslashes are duplicated, so that they act literally when included in a JavaScript template string", () => {
```

Comment:
Where did all these tests go??

Reply:
I have now brought them back in the form of compiler tests which check the result of compiling these against the full expected compiler output.